Sources/SafeDICore/Extensions/AttributeListSyntaxExtensions.swift

```diff
@@ -24,12 +24,7 @@ extension AttributeListSyntax {
 
     public var instantiableMacro: AttributeSyntax? {
         guard let attribute = first(where: { element in
-            switch element {
-            case let .attribute(attribute):
-                return IdentifierTypeSyntax(attribute.attributeName)?.name.text == InstantiableVisitor.macroName
-            case .ifConfigDecl:
-                return false
-            }
+            element.instantiableMacro != nil
```

Comment:
I simplified our macro access while I was adding associated types to the `Dependency.Source` enum in order to share this code.